### PR TITLE
rescaling and black primary

### PR DESCRIPTION
**Pull request by uberpye**
Following one of the paper referee's good ideas, I have made a few changes: 
I have re-scaled all plots that have the primary channel and another channel to be scaled to the primary channel and in the primary channel's units.
I have changed the primary channel's color to black and added an option to change the line width of the channel.

 [example](https://ldas-jobs.ligo-la.caltech.edu/~jeffrey.bidler/detchar/lasso-tests/lasso-test-suite/suitev0.10/RTD/)

I also removed 
from scipy.stats import spearmanr
because we don't use it anymore

from commit 2: added a line width option for auxiliary channels changed colormap to tab20 for more colors

### CODE REVIEW

**Comment by uberpye** [comment]
@duncanmmacleod this pull request is ready for your review.

**Comment by duncanmmacleod** [comment]
@macedo22, can you review this please?

File: bin/gwdetchar-lasso-correlation [hunk 1]
```diff
@@ -808,15 +844,35 @@ def generate_cluster(input_,):
...
+                    Table(data=(zip(*current_cluster)[3],
+                                zip(*current_cluster)[2]),
+                          names=('Channel', 'Pearson Coefficient')))
+                plot7_list = '%s_CLUSTER_LIST-%s.txt' % (
+                    re_delim.sub('_', str(currentchan))
+                            .replace('_', '-', 1),
+                    gpsstub)
+                clustertab.write(plot7_list, format='ascii', overwrite=True)
+                colormaps = [cmap(i) for i in
+                             numpy.linspace(0, 1, cluster_range+1)]
```
Comment (macedo22) [inline bin/gwdetchar-lasso-correlation]: @uberpye Just for conciseness, I think we could bring lines 838-846 / 850-858 outside the `if`/`else` so we only have that block once, but leave the `cluster_range` and `colormaps` initializations inside

**Comment by macedo22** [comment]
@uberpye Can you rebase this branch against `upstream/master` to include the newly commits?

File: bin/gwdetchar-lasso-correlation (hunk 1 above)
Comment (uberpye) [reply bin/gwdetchar-lasso-correlation]: sounds good, now done. as it is a small change a squashed it in to the earlier commit. now to rebase against upstream/master

**Comment by uberpye** [comment]
> 
> 
> @uberpye Can you rebase this branch against `upstream/master` to include the newly-added commits?

ok commits rebased against master!

**Comment by macedo22** [comment]
@uberpye it all looks good to me!

**Pull request merged**